src/nbsap/templates/aichi/aichi.html

```diff
@@ -0,0 +1,84 @@
+{% extends 'layout.html' %}
+
+{% load i18n %}
+{% load utils %}
+
+
+{% block title %}
+	NBSAP - {% trans "Aichi Goals" %}
+{% endblock %}
+
+
+{% block navmenu%}
+	{% include '_bits/nav_menu.html' %}
+{% endblock %}
+
+
+{% block sidebar %}
+	<div class="sidebar">
+		<h4 class="sidebar-title">Goals</h4>
+		<ul class="sidebar-menu">
+			{% for goal in goals %}
+				<li id="list-item" {% if goal.code == target.get_parent_goal.code %}class="active"{% endif %}>
+					<a>{% trans "Goal" %} {{ goal.code|upper }}</a>
+					<ul class="sidebar-submenu">
+						{% for current_target in goal.targets.all|sort_by_code %}
+							<li {% if current_target.code == target.code %}class="active"{% endif %}>
+								<a href="{% url 'goals' code=current_target.get_parent_goal.code aichi_target_id=current_target.id %}"
+									rel="tooltip" class="targets">
+									{% trans "Target" %} {{ current_target.code }}
```

Comment:
This is the perfect example that shows why we should use 2-space indentation in the templates. Half of the lines in this file are cropped and it makes it very difficult for me to actually review the code.